docs/modules/plugins/pages/plugin-web-app-to-rest-api.adoc

```diff
@@ -10,6 +10,39 @@ The plugin provides the integration between web application testing functionalit
 implementation(group: 'org.vividus', name: 'vividus-plugin-web-app-to-rest-api', version: '{current-version}')
 ----
 
+== Steps
+
+=== Check resources
+
+Check availability of resources on web pages
+
+Availability check logic:
+
+. If the `pages` row contains relative URL then it gets resolved against URL in `web-application.main-page-url` property
+. Gather elements by the CSS selector from each page
+. Get either `href` or `src` value from each element
+. For each received value execute https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/HEAD[HEAD] request
+.. If the status code is https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200[200 OK] then the check considered as passed
+.. If the status code if one of https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404[404 Not Found], https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405[405 Method Not Allowed], https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/501[501 Not Implemented], https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503[503 Service Unavailable] then GET request will be sent
+.. If the https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/GET[GET] status code is https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200[200 OK] then check considered as passed, otherwise failed
```

Comment:
```suggestion
.. If the https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/GET[GET] status code is https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200[200 OK] then check is considered as passed, otherwise failed
```

Reply:
check -> resource validation ?